libs/common/utils/geometry/esri/src/lib/common-utils-geometry-esri.ts

```diff
@@ -12,18 +12,17 @@ import { polygon as tPolygon } from '@turf/helpers';
  * @param feature Esri geometry
  */
 export function centroidFromGeometry(
-  feature: esri.Geometry | esri.Polygon | esri.Multipoint | esri.Point | esri.Polyline
-): Point;
-export function centroidFromGeometry(feature: any): Point {
-  if (feature.rings) {
+  feature: esri.Geometry | esri.Polygon | esri.Multipoint | esri.Point | esri.Polyline | Point
+): Point {
+  if ('rings' in feature) {
```

Comment:
Not exactly a functional change, but this can be used in TypeScript to scope the properties of a type